net/sockstats/sockstats.go

```diff
@@ -99,3 +100,11 @@ type LinkMonitor interface {
 func SetLinkMonitor(lm LinkMonitor) {
 	setLinkMonitor(lm)
 }
+
+// IsLikelyCellularInterface identifies whether a network interface is likely
```

Comment:
`net/interfaces.go` is likely a better home for this, if it's going to be public.

Could also not expose this function at all, and instead have the `SockStats` struct have a `CurrentInterfaceIsLikelyCellular` boolean field, which would also address the comment above about not having to loop over interfaces for every change.

Reply:
Done.  Added `SockStats.CurrentInterfaceCellular` (mirroring name on sockStats struct, but I can use the longer name if you prefer)